experimenter/experimenter/experiments/tests/test_changelog_utils.py

```diff
@@ -374,3 +375,21 @@ def test_generates_changelog_with_out_of_date_latest_change(self):
             experiment=experiment, experiment_data={"some_old": "data"}
         )
         generate_nimbus_changelog(experiment, self.user, "test message")
+
+    def test_generate_nimbus_changelog_skip_nonexistent_field(self):
+        non_existent_field_name = "nonexistent_field"
+        field_diff = {"old_value": "old_value", "new_value": "new_value"}
+        changelog = None
+        timestamp = None
+
+        try:
+            change_object = get_formatted_change_object(
+                non_existent_field_name,
+                field_diff,
+                changelog,
+                timestamp,
+            )
+        except Exception as e:
+            self.fail(f"Exception raised: {e}")
```

Comment:
TIL self.fail, but we dont need the try/except because if it throws the test will fail